_sass/content.scss

```diff
@@ -82,21 +82,16 @@
     }
   }
 
-  .task-list {
-    padding-left: 0;
-  }
-
   .task-list-item {
-    display: flex;
-    align-items: center;
-
     &::before {
       content: "";
     }
   }
 
   .task-list-item-checkbox {
     margin-right: 0.6em;
+    margin-left: -1.4em;
+    // The same margin-left is used above for ul > li::before
```

Comment:
Do we need to know this in order to use this? 
I suspect this information about something outside of this css can change at any moment, leaving us with a hanging 
 and outdated comment at some point.

Reply:
I think this is a helpful comment; both for maintainers (ex me) so we know why this magic number exists, and others who are extending this theme and may be confused as to why we have this very specific value. This fix comes from a requested user feature.

Reply:
That is fine, but know that is partially the reason we should write well written git commit messages 😄